nuxeo-ai-jaxrs/src/test/resources/mappings/nuxeo_datasets.json

```diff
@@ -6,7 +6,7 @@
"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"1-0","isTrashed":false,"title":"3 features, 23035 Training, 5759 Evaluation, Export id a4d503c3-5b44-448c-b335-e332087cb813","lastModified":"2019-05-25T07:50:31.302Z","facets":[]},{"entity-type":"document","repository":"default","uid":"84fa33c1-5cb3-4adf-9b9d-871d98448a02","path":"/Nuxeo Jira/datasets/e895c522-d0b9-40ce-9007-39dc2d8fba44","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"1-0","isTrashed":false,"title":"3 features, 25749 Training, 6437 Evaluation, Export id e895c522-d0b9-40ce-9007-39dc2d8fba44","lastModified":"2019-05-23T15:20:32.353Z","facets":[]},{"entity-type":"document","repository":"default","uid":"fec969c8-291b-4f9e-9df5-8fd6a9a4b92b","path":"/Nuxeo Jira/datasets/f2d6ea01-f07a-4ca4-a7e2-1d5e67e7de3d","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"1-0","isTrashed":false,"title":"4 features, 22764 Training, 5691 Evaluation, Export id f2d6ea01-f07a-4ca4-a7e2-1d5e67e7de3d","lastModified":"2019-05-25T07:50:00.180Z","facets":[]},{"entity-type":"document","repository":"default","uid":"7b769c6f-701d-4eba-bbc2-a3e2f2528ea2","path":"/Nuxeo Jira/datasets/6c1de6e7-752e-4d92-ad32-3f707badf3d2","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"3-0","isTrashed":false,"title":"Components+Tags","lastModified":"2019-05-23T16:49:39.421Z","facets":[]},{"entity-type":"document","repository":"default","uid":"cd53677c-72ba-4531-a3b4-b66d4715ec5b","path":"/Nuxeo Jira/datasets/fb4a4505-90c6-4cc2-b1c6-e58c853fd148","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"4-0","isTrashed":false,"title":"T+D>Creator","lastModified":"2019-05-25T07:59:07.601Z","facets":[]},{"entity-type":"document","repository":"default","uid":"6c9b51a7-9cd7-449d-85e2-8e9a560e47b6","path":"/Nuxeo Jira/datasets/c1807187-c3d6-493d-85a5-fd7ddfb3ba73","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"3-0","isTrashed":false,"title":"Versions","lastModified":"2019-05-24T22:01:48.709Z","facets":[]},{"entity-type":"document","repository":"default","uid":"a3660985-5ece-41fd-9dfc-4f5e6e43975a","path":"/Nuxeo Jira/datasets/dd6e397b-9935-4b89-8202-cb32c4beaf54","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"3-0","isTrashed":false,"title":"title+description+components","lastModified":"2019-05-23T15:54:15.994Z","facets":[]}]},
```

Comment:
I think this should be `dataset_export`

Reply:
no, it's a type not a schema

Reply:
we had AI_Corpus type that conforms to `ai_corpus` schema. Now we will have `DatasetExport` type that conforms to `dataset_export` schema